devtools/index.js

```diff
@@ -1,4 +1,4 @@
-import { initDevTools } from './devtools';
-
-initDevTools();
-
+if (NODE_ENV !== 'production') {
```

Comment:
@t47io yes, it should be `process.env.NODE_ENV`. Not sure about using outside of a bundler.  @developit are builds for CDN somehow separate we could feed `process.env.NODE_ENV` into rollup there?

> I was thinking process.env.NODE_ENV && process.env.NODE_ENV !== 'production'

Well, in ideal case it should be `typeof process !== 'undefined' && process.env && process.NODE_ENV !== 'production'` but I not sure how `webpack` would handle that.